File: app/src/main/java/me/devsaki/hentoid/json/core/JsonAchievements.java
package me.devsaki.hentoid.json.core;

import java.util.List;

import me.devsaki.hentoid.database.domains.Achievement;

public class JsonAchievements {

    public List<JsonAchievement> achievements;

    public static class JsonAchievement {
        public int id;
        public Achievement.Type type;
    }
}

File: app/src/main/java/me/devsaki/hentoid/json/core/JsonAchievements.kt
package me.devsaki.hentoid.json.core

import me.devsaki.hentoid.database.domains.Achievement

data class JsonAchievements(
    val achievements: List<JsonAchievement>
) {
    data class JsonAchievement(
        val id: Int,
        var type: Achievement.Type
    )
}
